modules/network-observability-cli-capturing-metrics.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/network_observability/netobserv_cli/netobserv-cli-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-cli-capturing-metrics_{context}"]
+= Capturing metrics
+You can generate on-demand dashboards in Prometheus by using a service monitor for Network Observability.
+
+.Prerequisites
+* Install the {oc-first}.
+* Install the Network Observability CLI (`oc netobserv`) plugin.
+
+.Procedure
+. Capture metrics with filters enabled by running the following command:
++
+.Example output
+[source,terminal]
+----
+$ oc netobserv metrics --enable_filter=true --cidr=0.0.0.0/0 --protocol=TCP --port=49051
+----
+. Open the link provided in the terminal to view the *NetObserv / On-Demand* dashboard:
++ 
+.Example URL
+[source,terminal]
+----
+https://console-openshift-console.apps.rosa...openshiftapps.com/monitoring/dashboards/netobserv-cli
+----
++
+[NOTE]
+====
+Features that are not enabled present as empty graphs.
```

Comment:
I am not sure what this means but I am guessing it makes sense to the intended audience :)

Reply:
Yeah, I think it should make sense. Network Observability has some features that are widely known in the docs and use-case-wise, but optional to turn on/off. So if you have any of these features off, then you get empty graphs.